addons/skin.estuary/language/resource.language.en_gb/strings.po

```diff
@@ -743,3 +743,9 @@ msgstr ""
 msgctxt "#31163"
 msgid "Show Fanart background"
 msgstr ""
+
+#: This string is used at the skin settings for a switch to either show the profile name next to the clock or not
```

Comment:
If somebody rearranges controls, you might have to change the comment to fit the new location of the label  - and I bet this will be forgotten. I would avoid those "next to something" comments. Describe the item itself, not how it is currently related to something else.

Reply:
since this is a contextual note it has to start with `#.` and not `#:`.
As for the string, how about
`#. Label of a setting that, if enabled, always shows the name of the currently logged-in profile in the UI`